src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -120,7 +123,20 @@ public void TestEscapeOnce()
         }
 
         [Test]
-        public void TestTruncateWords()
+        public void TestTruncateWordsV20()

```

Comment:
Can we split these into LegacyFilterTests?

Reply:
Please see new structure for version specific tests. It adds more tests (every version is tested) and is limited to unit tests only. It's WIP.